### PR TITLE
updating some URLs

### PR DESCRIPTION
**Pull request by taw00**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ are carried out collectively by the network. Dash Core is the name of open
 source software which enables the use of this currency.
 
 For more information, as well as an immediately useable, binary version of
-the Dash Core software, see https://www.dashpay.io/downloads.
+the Dash Core software, see https://www.dash.org/downloads.
```
Comment (UdjinM6) [inline README.md]: This patch is incomplete - doesn't fix line number 4.
And it's already fixed in 12.1 btw.

File: INSTALL [hunk 2]
```diff
@@ -7,7 +7,7 @@ Use the autogen script to prepare the build environment.
     make
 
 Precompiled binaries are available at github, see
-https://github.com/dashproject/dash-binaries
+https://github.com/dashpay/dash/releases
```
Comment (UdjinM6) [inline INSTALL]: That's not true actually, `.../releases` only have zipped/tared sources. Correct URL for precompiled binaries is https://github.com/dashpay/dash-binaries
This needs to be fixed in 12.1 too.

**Comment by UdjinM6** [comment]
Thanks for contributing!

However there are some issues with these changes, see line comments.

Another thing to note: we don't merge PRs into master branch, we use dev branches for this. Please submit to `v0.12.0.x` (stable/current version) and `v0.12.1.x` (test/dev version) branches only.

**Comment by taw00** [comment]
Hey. Okay. (a) Can't believe my eyes just swooped right over the other URL. And (b) ... I haven't looked at the dev branches yet, but maybe the README.md really needs to be updated to mention that. Of course, there are a number of TODOs in there... as there always are.

Thanks for kindly redirecting me. I will fork from a different branch. :)

**Pull request closed without merging**